### PR TITLE
test(rolling_upgrade): rerunning a short rolling upgrade

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
due to https://github.com/scylladb/scylla/issues/5840 I'm rerunning a short version of the test,
only an upgrade of one node.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: upgrade_test.py [hunk 1]
```diff
@@ -456,36 +456,36 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
             # write workload during entire test
```
Comment (roydahan) [inline upgrade_test.py]: Remove the two lines above:
self.prepare_keyspaces_and_tables()
            self.fill_and_verify_db_data('BEFORE UPGRADE', pre_fill=True)	            self.fill_and_verify_db_data('BEFORE UPGRADE', pre_fill=True)

File: upgrade_test.py [hunk 2]
```diff
@@ -503,7 +510,7 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
                 self.db_cluster.node_to_upgrade.check_node_health()
 
                 # wait for the prepare write workload to finish
-                self.verify_stress_thread(prepare_write_cs_thread_pool)
+                # self.verify_stress_thread(prepare_write_cs_thread_pool)
 
                 # read workload (cl=QUORUM)
                 self.log.info('Starting c-s read workload (cl=QUORUM n=10000000)')
```
Comment (roydahan) [inline upgrade_test.py]: remove

File: upgrade_test.py [hunk 3]
```diff
@@ -503,7 +510,7 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
                 self.db_cluster.node_to_upgrade.check_node_health()
 
                 # wait for the prepare write workload to finish
-                self.verify_stress_thread(prepare_write_cs_thread_pool)
+                # self.verify_stress_thread(prepare_write_cs_thread_pool)
```
Comment (roydahan) [inline upgrade_test.py]: remove everything below this

**Comment by bentsi** [comment]
Is it still a draft?

**Comment by ShlomiBalalis** [comment]
@bentsi it is, you can ignore this pr

File: upgrade_test.py [hunk 4]
```diff
@@ -442,36 +442,26 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
...
+        # target_upgrade_version = self.params.get('target_upgrade_version', default='')
+        # self.truncate_entries_flag = False
+        # if target_upgrade_version and parse_version(target_upgrade_version) >= parse_version('3.1') and \
+        #         not is_enterprise(target_upgrade_version):
+        #     self.truncate_entries_flag = True
+
+        # write workload during entire test
+        # self.log.info('Starting c-s write workload during entire test')
+        # write_stress_during_entire_test = self.params.get('write_stress_during_entire_test')
+        # self.run_stress_thread(stress_cmd=write_stress_during_entire_test)
```
Comment (roydahan) [inline upgrade_test.py]: This line is a MUST

**Comment by bentsi** [comment]
@ShlomiBalalis why do we need it? is it in progress? Is it intended to be merged someday?

**Comment by ShlomiBalalis** [comment]
No, it's not necessary anymore, I'll close it

**Pull request closed without merging**